index.js

```diff
@@ -3,26 +3,43 @@
 var crypto = require('crypto');
 var util = require('util');
 var properties = require('./properties.json');
+var uuid = require('uuid');
 
 var algCryptoMap = {
-  HS256: 'sha256',
-  HS384: 'sha384',
-  HS512: 'sha512',
-  RS256: 'RSA-SHA256'
+  HS256: 'SHA256',
+  HS384: 'SHA384',
+  HS512: 'SHA512',
+  RS256: 'RSA-SHA256',
+  RS384: 'RSA-SHA384',
+  RS512: 'RSA-SHA512',
+  ES256: 'RSA-SHA256',
```

Comment:
Node's crytpo library has strange names for things, hence this map.  There are tests with ES keypairs that assert the correct behavior.